### ISSUE TITLE
Creating new profiles may delete config file

### ISSUE THREAD

**Issue by nevotheless** [Eugeny/tabby#4506]
Unfortunately this just happend to me and i didn't have any backups of my config file. So i'm sitting here 😭 sobbing over my lost profiles.

Anyhow, i want to help that this doesn't happen to anyone else and report this here. 

**What i did**
Creating a new profile
- with a name
- with a group name
- with a host
- with a username
- with a password set via "auto" authentication method

I did this couple of times today and it removed my config once and one other time briefly (like for a split second) created the new profile with the new group in the profiles menu and then deleted that as well. Other times this worked like it was supposed to so the behavior seems pretty random for me at this moment.

The config file got completely wiped, including application and hotkey settings only the vault stayed intact with all credentials. 

**Environment**
Windows 10 20H2 (19042.1165)
Tabby Alpha 1.0.152

Tabby install location: `C:\Users\myDomainUser\AppData\Roaming\tabby`

**Comment by Eugeny** [comment]
Hey, sorry about the profiles 😢  Was there a `config.json.new` file present right after the config got killed? I'm writing the config there first and then renaming it into `config.json` atomically. Also check if disabling Defender realtime protection temporarily "fixes" the issue.

**Comment by nevotheless** [comment]
I didn't see a config.json.new file there only the config.json. (I checked ~10 minutes after the file got removed)
After a restart Tabby seems to work properly again. When i create or duplicate a session now, it doesn't "twitch" anymore after saving the session.

I'm using Tabby at work, we're not using Windows Defender but Kaspersky. When this happens again and i'm able to replicated this issue, i'll try with Kaspersky stopped to verify this.

However maybe it's related to this "twitching" whenever Tabby tries to do anything with profiles. Like it was here as well: #4330
